plugins/generator-addon-1.16.x/addon-1.16.x/templates/block/block_ore_feature.json.ftl

```diff
@@ -1,18 +1,28 @@
 <#-- @formatter:off -->
 <#include "../mcitems.ftl">
 {
-    "format_version": "1.13.0",
-    "minecraft:ore_feature": {
-      "description": {
-        "identifier": "${modid}:${modid}_${registryname}_ore_feature"
-      },
-      "count": ${data.frequencyOnChunk},
-      "places_block": "${modid}:${registryname}",
-      "may_replace": [
-        <#list data.blocksToReplace as block>
-        "${mappedMCItemToIngameNameNoTags(block)}"<#if block?has_next>,</#if>
-        </#list>
-      ]
-    }
+	"format_version": "1.17.0",

```

Comment:
Don't use the 1.17 version for a 1.16.x generator.

Reply:
you cannot make add-ons for 1.16 anymore cause you cannot go backwards on Minecraft bedrock